frontend/packages/knative-plugin/src/components/traffic-splitting/__tests__/TrafficSplitting.spec.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import * as Renderer from 'react-test-renderer';
+import TrafficSplitting from '../TrafficSplitting';
+import {
+  mockRevisions,
+  mockServiceData,
+} from '../../../utils/__mocks__/traffic-splitting-utils-mock';
+
+describe('TrafficSplitting', () => {
```

Comment:
nit:
```suggestion
describe(TrafficSplitting.displayName, () => {
```

Reply:
@sahil143 unless we explicitly assign a display name, this won't work.

Reply:
@christianvogt This works with the functional component as well without defining it explicitly.

Reply:
Never Mind, This doesn't work, just tested this.